mvz/methods/shared.py

```diff
@@ -10,6 +10,11 @@
 # to the left.
 min_value_x = 100
 
+Frame = Tuple[float, float, float, float, float]
+Frames = List[Frame]
+FrameSpecOutput = Union[List[const.BoundingBox],
+                        Frames]
```

Comment:
I think this may not type check any more, since there's some other signatures, like the `shared.tuple4` I had to use because of imperfect type inference, that will provide `int` instead of `float` for the last 4 elements of the frame.  I think that just moving this back to its old definition instead of using `Frames` in it should work?